test-clients/src/main/java/com/hedera/services/yahcli/commands/accounts/SendCommand.java

```diff
@@ -71,11 +71,20 @@ public class SendCommand implements Callable<Integer> {
 
     @Override
     public Integer call() throws Exception {
+        // read data from config.yml
```

Comment:
This is another place we'd have to keep track of the config filename. I'm fine with leaving this comment in if it's helpful though

Reply:
I will remove it not an issues